samples/bot-conversation-sso-quickstart/js/bots/teamsBot.js

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const { DialogBot } = require('./dialogBot');
+const { tokenExchangeOperationName } = require('botbuilder');
+const { SsoOauthHelpler } = require('../ssoOauthHelpler');
+
+class TeamsBot extends DialogBot {
+    /**
+     *
+     * @param {ConversationState} conversationState
+     * @param {UserState} userState
+     * @param {Dialog} dialog
+     */
+    constructor(conversationState, userState, dialog) {
+        super(conversationState, userState, dialog);
+        this._ssoOauthHelper = new SsoOauthHelpler(process.env.connectionName, conversationState);
+
+        this.onMembersAdded(async (context, next) => {
+            const membersAdded = context.activity.membersAdded;
+            for (let cnt = 0; cnt < membersAdded.length; cnt++) {
```

Comment:
Maybe we can use for...of instead of manually using cnt?
```
for (const member of membersAdded) {
  if (member.id !== ...) {
  }
}
```

Reply:
I think it's ok